File: ios/semomun/Home/RNTHomeVCManager.m
//
//  RNTHomeVCManager.m
//  semomun
//
//  Created by 신영민 on 2022/07/21.
//

#import <React/RCTBridgeModule.h>
#import <React/RCTViewManager.h>
#import "semomun-Swift.h"

@interface RNTHomeVCManager : RCTViewManager
@end

@implementation RNTHomeVCManager

RCT_EXPORT_MODULE()

- (UIView *)view
{
  return [[HomeVCAdapterView alloc] init];
}


+ (BOOL)requiresMainQueueSetup
{
  return true;
}

@end

File: ios/semomun/Home/RNTHomeVCManager.swift
//
//  HomeVCManager.swift
//  semomun
//
//  Created by 신영민 on 2022/07/22.
//

import UIKit


@objc(RNTHomeVCManager)
class RNTHomeVCManager : RCTViewManager {
  
  override func view() -> UIView! {
    return HomeVCAdapterView()
  }
    
}
